src/hotspot/share/compiler/compilerOracle.cpp

```diff
@@ -776,7 +777,13 @@ static void scan_value(enum OptionType type, char* line, int& total_bytes_read,
         }
       }
 #ifndef PRODUCT
```

Comment:
Missing `#ifdef COMPILER2` for this and `PrintIdealPhase`.

Reply:
done!